codechain/crypto/hash.py

```diff
@@ -0,0 +1,73 @@
+import hashlib
+
+
+def blake256(data):
+    if isinstance(data, str):
+        data = bytes.fromhex(data)
+
+    h = hashlib.blake2b(digest_size=32)
+    h.update(data)
+    return h.hexdigest()
```

Comment:
I think returning `bytes` type with `h.digest()` more useful. 
It can be directly fed into functions that require `byteslike` type (like `rlp`, etc. If you use string, there should be a special care). 
If you want to convert it to string, then `blake256(data).hex()` can be used. It is more concise and doesn't interrupt keyboard flowing like `blake256(data) -> bytes.fromhex(blake256(data))`. Just append `.hex()`. We can leverage existing `class Hxxx` that extends bytes type. `H256(blake256(data))` would just require memcpy operation. 
If you want to know its size, then just `len(blake256(data))` would return byte size, not `len(blake256(data))/2` It doesn't make confusion in its output format (does it have '0x' prefix? All hex alphabets are lowercase? uppercase? What is the case of '0x'?)

Reply:
I agree.